### ISSUE TITLE
Needs more doge

### ISSUE THREAD

**Issue by brettgoulder** [heroku/rack-timing#1]

**Comment by rhyselsmore** [comment]
+1

**Comment by chadbailey59** [comment]
![doge-1387429157](https://f.cloud.github.com/assets/228358/2143017/a3c72d8c-936c-11e3-9bc6-1ee1372d49b8.gif)

**Comment by rhyselsmore** [comment]
I am noting that this was not addressed, and simply closed without explanation.

**Comment by chadbailey59** [comment]
I don't think this was adequately addressed with my addition of a single doge. I'll add a few more, for posterity's sake:

![doge](https://camo.githubusercontent.com/23b3884dc37327177482e862e53d72c5541add2d5f0153afa4d27297b010115d/68747470733a2f2f662e636c6f75642e6769746875622e636f6d2f6173736574732f3232383335382f323134333031372f61336337326438632d393336632d313165332d396263362d3165653133373264343962382e676966) ![doge](https://camo.githubusercontent.com/23b3884dc37327177482e862e53d72c5541add2d5f0153afa4d27297b010115d/68747470733a2f2f662e636c6f75642e6769746875622e636f6d2f6173736574732f3232383335382f323134333031372f61336337326438632d393336632d313165332d396263362d3165653133373264343962382e676966) ![doge](https://camo.githubusercontent.com/23b3884dc37327177482e862e53d72c5541add2d5f0153afa4d27297b010115d/68747470733a2f2f662e636c6f75642e6769746875622e636f6d2f6173736574732f3232383335382f323134333031372f61336337326438632d393336632d313165332d396263362d3165653133373264343962382e676966)
